app/src/main/java/edu/mit/mitmobile2/shuttles/ShuttlesActivity.java

```diff
@@ -159,13 +159,15 @@ public void updateShuttleRoutesStopView(final MITShuttle mitShuttle, final boole
                 stopParams, null, new Callback<MITShuttleStopWrapper>() {
                     @Override
                     public void success(MITShuttleStopWrapper mitShuttleStopWrapper, Response response) {
-                        if (mitShuttleStopWrapper.getPredictions().size() != 0) {
-                            if (isFirstStop) {
-                                mitShuttle.setFirstMinute(mitShuttleStopWrapper.getPredictions().get(0).
-                                        getSeconds() / 60 + "m");
-                            } else {
-                                mitShuttle.setSecondMinute(mitShuttleStopWrapper.getPredictions().get(0).
-                                        getSeconds() / 60 + "m");
+                        if (mitShuttleStopWrapper.getPredictions() != null) {
```

Comment:
Instead of nesting the IFs, you can check if predictions is null and predictions size >0 in 1 IF condition, as long as the null check is first